LICENSE

```diff
@@ -186,7 +186,7 @@ APPENDIX: How to apply the Apache License to your work.
    same "printed page" as the copyright notice for easier
    identification within third-party archives.
 
-Copyright [yyyy] [name of copyright owner]
+Copyright 2008-2015 Chef Software, Inc.
```

Comment:
This shouldn't change. We don't claim copyright for anyone's contribution that doesn't work for us, so the template shouldn't encourage them to put our name there.